### PR TITLE
add eslint

### PR DESCRIPTION
**Pull request by oaguinaga**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "rules": {
+    "eqeqeq": "off",
+    "indent": ["error", 2]
```
Comment (cakuki) [inline .eslintrc.js]: I think this indentation change is unnecessary. Given that autofix cannot actually fix JSDoc comment block indentations. So either all comment blocks should be fixed or this indent rule should be left to default `eslint:recommended` to 4 spaces.

Comment (dashersw) [reply .eslintrc.js]: ohhh :( well — i'm not sure. what do you think? should we leave it at 4, then? or update the comments as well?

Comment (oaguinaga) [reply .eslintrc.js]: what's the decision for this, should I change to 4 or update the comments?

Comment (dashersw) [reply .eslintrc.js]: let's make it 4, for now.

**Pull request merged**